data/game_teams_dummy.csv

```diff
@@ -0,0 +1,37 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
+2017030223,15,away,LOSS,REG,Barry Trotz,2,5,41,12,4,1,39,4,7
+2017030223,5,home,WIN,REG,Mike Sullivan,3,5,52,12,4,1,61,9,6
+2017030224,15,away,LOSS,REG,Barry Trotz,1,5,43,13,3,1,50,5,3
+2017030224,5,home,WIN,REG,Mike Sullivan,3,6,25,11,4,2,50,11,5
```

Comment:
Making sure to get both matching game_ids was a good call! Might lead to some issues if we were missing one half of the game_ids for a corresponding game.